pmm/aws-staging-start-pmm3.groovy

```diff
@@ -0,0 +1,486 @@
+                                set -o errexit
+                                set -o xtrace
+                                docker exec ${VM_NAME}-server bash -c "echo exclude=mirror.es.its.nyu.edu | tee -a /etc/yum/pluginconf.d/fastestmirror.conf"
+                                docker exec ${VM_NAME}-server yum update -y percona-release
+                                docker exec ${VM_NAME}-server sed -i'' -e 's^/release/^/experimental/^' /etc/yum.repos.d/pmm2-server.repo
+                                docker exec ${VM_NAME}-server percona-release enable percona experimental
+                                docker exec ${VM_NAME}-server yum clean all
+                            """
+                        }
+                    }
+                }
+            }
+        }
+        stage('Run Clients') {
+            steps {
+                node(env.VM_NAME){
+                    setupPMMClient(SERVER_IP, CLIENT_VERSION.trim(), PMM_VERSION, ENABLE_PULL_MODE, ENABLE_TESTING_REPO, CLIENT_INSTANCE, 'aws-staging', ADMIN_PASSWORD)
+                    script {
+                        env.PMM_REPO="experimental"
+                        if(env.CLIENT_VERSION == "pmm2-rc") {
+                            env.PMM_REPO="testing"
+                        }
+                        
+                    }
+                    sh '''
+                        set -o errexit
+                        set -o xtrace
+                        export PATH=$PATH:/usr/sbin
+                        export PMM_CLIENT_VERSION=${CLIENT_VERSION}
+                        if [[ ${CLIENT_VERSION} == 3-dev-latest ]]; then
+                                export PMM_CLIENT_VERSION="latest"
+                        fi
+                        [ -z "${CLIENTS}" ] && exit 0 || :
+
+                        if [[ ${PMM_VERSION} == pmm2 ]]; then
+
+                            export PMM_SERVER_IP=${SERVER_IP}
+
+                            if [[ ${CLIENT_VERSION} != 3-dev-latest ]]; then
+                                export PATH="`pwd`/pmm2-client/bin:$PATH"
+                            fi
+                            if [[ ${CLIENT_INSTANCE} == no ]]; then
+                                export PMM_SERVER_IP=${IP}
+                            fi
+
+                            bash /srv/pmm-qa/pmm-tests/pmm-framework.sh \
+                                --ms-version  ${MS_VERSION} \
+                                --mo-version  ${MO_VERSION} \
+                                --ps-version  ${PS_VERSION} \
+                                --modb-version ${MODB_VERSION} \
+                                --md-version  ${MD_VERSION} \
+                                --pgsql-version ${PGSQL_VERSION} \
+                                --pxc-version ${PXC_VERSION} \
+                                --pdpgsql-version ${PDPGSQL_VERSION} \
+                                --download \
+                                ${CLIENTS} \
+                                --pmm2 \
```

Comment:
@vyarosh @puneet0191 do you think it should be `pmm` or `pmm3`?

Reply:
no need for a new job for staging start, we will have a new stage just for pmm3, unless their will be a significant changes in the way pmm2, pmm3, pmmX are launched.

Reply:
@BupycHuk @puneet0191 @surbhat1595 

I agree that ideally we would not need yet another pipeline to stage AWS instances. However, for the sake of code isolation between versions and ease of maintenance, I suggest we create a separate branch. 

I spent hours to make the staging pipeline work until I got a satisfactory result - [check this PR](https://github.com/Percona-Lab/jenkins-pipelines/pull/2393/files) and [this pipeline](https://pmm.cd.percona.com/view/DevOps/job/aws-staging-start-pmm3-ad). Unfortunately, there are still things to improve and simplify, which will take time to get right. 

Therefore, at this point in time it's much easier to go ahead with two pipelines, keeping in mind that we can merge them anytime we feel confident we no longer need two.